src/mongocxx/bulk_write.cpp

```diff
@@ -33,8 +33,16 @@ bulk_write& bulk_write::operator=(bulk_write&&) noexcept = default;
 
 bulk_write::~bulk_write() = default;
 
-bulk_write::bulk_write(bool ordered)
-    : _impl(stdx::make_unique<impl>(libmongoc::bulk_operation_new(ordered))) {
+bulk_write::bulk_write(options::bulk_write options)
+    : _impl(stdx::make_unique<impl>(libmongoc::bulk_operation_new(options.ordered()))) {
+    auto options_wc = options.write_concern();
```

Comment:
Is this necessary because options.write_concern() returns an optional? Is there any problem with the way options.ordered() is used above, since it also returns an optional?